src/Microsoft.DotNet.Build.Tasks/PackageFiles/Build.Common.targets

```diff
@@ -8,6 +8,10 @@
 
   <PropertyGroup>
     <BuildToolsTaskDir Condition="'$(BuildToolsTaskDir)' == ''">$(ToolsDir)</BuildToolsTaskDir>
+    
+    <!-- For UAP, we need to bin place the resources as resw files to create the runner's pri file. -->
+    <ResourcesFolderPath Condition="'$(ResourcesFolderPath)' == ''">$(RuntimePath)resw</ResourcesFolderPath>
+    <CommonPriFile Condition="'$(CommonPriFile)' == ''">$(RuntimePath)resources.pri</CommonPriFile>
```

Comment:
can we store this file under sub-folder like "PRI" or even with the rest of the rew files?